### PR TITLE
Feature: Move from bsb to Rescript

### PR DESCRIPTION
**Pull request by gensjaak**

### CODE REVIEW

File: examples/DecimalsMatters.res [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// Initialize a new big instance
+let decimal = 201.57512->big
+
+// Fix an output
+Js.log(decimal)
+Js.log(decimal->toFixed(2))
+
+// Use the operators
+Js.log(0.1->big +. 0.2->big)
+Js.log(2.5->big *. 0.->big)
```
Comment (gensjaak) [inline examples/DecimalsMatters.res:13]: @LeoLeBras Tu te rappelles du besoin d'avoir cet exemple à côté ?

**Review by gensjaak — COMMENTED** [review]
LGTM @LeoLeBras

File: examples/DecimalsMatters.res (hunk 1 above)
Comment (LeoLeBras) [reply examples/DecimalsMatters.res:13]: Pas vraiment non , mais c'est toujours pratique d'avoir un exemple !

File: __tests__/Big_spec.res [hunk 2]
```diff
@@ -0,0 +1,54 @@
+open Jest
+open Big
+open! Big.Operators
+
+describe("Big.js bindings from ReasonML / Ocaml", () => {
```
Comment (LeoLeBras) [inline __tests__/Big_spec.res:5]:
```suggestion
describe("Big.js bindings from ReScript", () => {
```

File: README.md [hunk 3]
```diff
@@ -1,62 +1,62 @@
...
 
 /* or */ let decimal = 2020.2065->big;
 ```
 
 Fix an output
 
-```reason
+```rei
 Js.log(decimal $. 2); // output -> 2020.21
 ```
```
Comment (LeoLeBras) [inline README.md:55]: it doesn't exist anymore it seems
read the readme again :)

File: __tests__/Big_spec.res [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+  test("should compare w/ eq", () => expect(79.11->big->eq(79.10->big)) === false)
+
+  test("should compare w/ eq", () => expect(0.->big->eq(0.0000000000001->big)) === false)
+
+  test("should compare w/ eq after round", () =>
+    expect(0.->big->eq(0.0000000000001->big->round(1))) === true
+  )
+
+  test("should compare w/ gt", () => expect(79.11->big->gt(79.10->big)) === true)
```
Comment (LeoLeBras) [inline __tests__/Big_spec.res:39]:
```suggestion
  test("should compare with gt", () => expect(79.11->big->gt(79.10->big)) === true)
```

what is "/w" ?

**Review by LeoLeBras — CHANGES_REQUESTED** [review]

File: __tests__/Big_spec.res (hunk 4 above)
Comment (gensjaak) [reply __tests__/Big_spec.res:39]: with

File: src/Big.res [hunk 5]
```diff
@@ -0,0 +1,134 @@
+@genType
+type t = float
```
Comment (LeoLeBras) [inline src/Big.res:2]:
```suggestion
type t
```

as this is a specific type of the `big.js` library, I would advise doing this

**Review by LeoLeBras — CHANGES_REQUESTED** [review]

File: package.json [hunk 6]
```diff
@@ -40,9 +41,9 @@
     "@types/jest": "25.1.4",
     "@types/node": "13.9.1",
     "babel-jest": "25.1.0",
-    "bs-platform": "9.0.0",
-    "gentype": "3.42.0",
+    "gentype": "^4.1.0",
```
Comment (LeoLeBras) [inline package.json:44]:
```suggestion
    "gentype": "4.1.0",
```

**Review by LeoLeBras — CHANGES_REQUESTED** [review]

File: package.json (hunk 6 above)
Comment (gensjaak) [reply package.json:44]: Why ?

Comment (LeoLeBras) [reply package.json:44]: to be sure that the installed version will be strictly the one defined in the package.json
and avoid being on the wrong version by mistake

Comment (gensjaak) [reply package.json:44]: Okay I guessed it... so do the others ?

Comment (LeoLeBras) [reply package.json:44]: yes !

**Review by LeoLeBras — APPROVED** [review]

**Pull request merged**